api/cems.py

```diff
@@ -0,0 +1,59 @@
+import sys
+import socket
+# import iota
+import datetime
+import requests
+import json
+import hashlib
+import base64
+
+
+def get_data(Tx, data):
```

Comment:
Do you think is it possible to let this function to become `utils` func ?

Reply:
This function is used to check bems's data's hash is same as data on IOTA or not.
Who's `utils` function ? PyOTA's?

Reply:
Yes, it is.

Reply:
Resolve in 4c34f06.